assets/src/components/ui/drop-down-menu/index.tsx

```diff
@@ -21,7 +23,7 @@ const DropDownMenu = ( {
 	tooltipPosition = 'top left',
 	index = 0,
 	...otherProps
-} ) => {
+} : PropTypes.InferProps<typeof DropDownMenu.propTypes> ) => {
```

Comment:
this is going to conflict with my branch that changes the folder structure...
although I might have to rework that branch to avoid having a gazillion broken unit tests { : \

Reply:
Not a problem.
the power of sublime merge will help me :hammer_and_wrench:

Reply:
@tn3rb I had reverted this change since it was causing tests failure and orchestrating all the stuff to handle `tsx` for `jest` needs to be done in a separate PR.
***
The only thing left for now in this PR is docs, as suggested by @mnelson4